manual/manual/cmds/unified-options.etex

```diff
@@ -269,14 +269,39 @@ but before the standard library directory. See also option "-nostdlib".
 
 If the given directory starts with "+", it is taken relative to the
 standard library directory.  For instance, "-I +unix" adds the
-subdirectory "unix" of the standard library to the search path.
+subdirectory "unix" of the standard library to the load path.
```

Comment:
(just looking at the documentation) What is the semantics of `-I <path>` or `-inc <path>` when `<path>` is itself a relative path? I would assume that they are resolved from the current working directory in which the compiler is invoked (and in particular, not within all directories in the load path, which would lead to a combinatorial explosion)? Is this specified explicitly somewhere?

Reply:
Indeed, this is the current semantics; it doesn't look like it is explicitly documented anywhere.